Project Walking/Source code/app/src/main/java/com/teamapricot/projectwalking/MainActivity.java

```diff
@@ -1,38 +1,41 @@
 package com.teamapricot.projectwalking;
 
-import androidx.appcompat.app.AppCompatActivity;
-
+import android.app.NotificationChannel;
+import android.app.NotificationManager;
+import android.os.Build;
 import android.os.Bundle;
 import android.view.View;
-import android.widget.TextView;
+import android.widget.Button;
 
-import com.teamapricot.projectwalking.handlers.CameraHandler;
-import com.teamapricot.projectwalking.photos.PhotoController;
+import androidx.appcompat.app.AppCompatActivity;
 
 public class MainActivity extends AppCompatActivity {
-    private PhotoController photoController;
-    LocationHandler locationHandler;
+
+    Button notifyBtn;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-
-        PermissionHandler permissionHandler = new PermissionHandler(this);
-        CameraHandler cameraHandler = new CameraHandler(this);
-
-        photoController = new PhotoController(this, permissionHandler, cameraHandler);
-
         setContentView(R.layout.activity_main);
-
-        locationHandler = new LocationHandler(this, 2000);
-
-        locationHandler.registerUpdateListener(position -> {
-            ((TextView)findViewById(R.id.helloWorld))
-                    .setText("(" + position.getLatitude() + "," + position.getLongitude() + ")");
+        createChannel();
+        notifyBtn = findViewById(R.id.notify_button);
```

Comment:
This button with id `R.id.notify_button` currently does not exist in "activity_main.xml". Maybe it disappeared when you merged the main branch with the current branch? Either way, add it back, otherwise the code does not compile.